CRM/Report/Form/Case/Summary.php

```diff
@@ -277,6 +277,13 @@ public function from() {
     $crt = $this->_aliases['civicrm_relationship_type'];
     $ccc = $this->_aliases['civicrm_case_contact'];
 
+    foreach ($this->_columns['civicrm_relationship']['filters'] as $fieldName => $field) {
+      if (!empty($this->_params[$fieldName . '_op'] && isset($this->_params[$fieldName . '_value']))) {
```

Comment:
@chamilwijesooriya I think test is failing because the above line misses a closing brace for the !empty() condition. Can you try replacing it with -

    if (!empty($this->_params[$fieldName . '_op']) && isset($this->_params[$fieldName . '_value'])) {

Reply:
done thanks for that